scripts/create_synthetic_population.R

```diff
@@ -0,0 +1,92 @@
+## =================================#
+## Setup ---------------
+## =================================#
+library(tidyverse)
+library(tigris)
+library(tidycensus)
+
+state_synth <- "WY"
+year_synth <- 2023
+population_size <- 100
+
+## =================================#
```

Comment:
We should probably have a fixed seed here to have a consistent synthetic population